src/kernel/include/shared/consts.h

```diff
@@ -31,8 +31,11 @@
 #define MAX_SERVICE_DEPTH 31
 #define MAX_NUM_THREADS (64 * NUM_CPU)
 
-/* Stacks are 2 * page_size (expressed in words) */
-#define MAX_STACK_SZ_BYTE_ORDER 12
+/*
+ * Stacks are 4 * page_size (expressed in words) 
+ * Thus, usually this would be 4 * 4k = 16k bytes
+ */
+#define MAX_STACK_SZ_BYTE_ORDER 14
```

Comment:
I had no idea we made them so large. That's huge! Did you need to increase the size for the purposes of DPDK?

Reply:
Yes. DPDK requires a larger stack size.